File: codekata/absolute beginner/766.js
/*
The area of an equilateral triangle is ¼(√3a2) where "a" represents a side of the triangle. 
You are provided with the side "a". Find the area of the equilateral triangle.

Positive TestCase :
    Input-20
    Output-173.21
*/

function fun(x){
    var a=(1.73205080757/4)*(x**2)
    console.log(a.toFixed(2))
}

fun(20)

File: codekata/absolute beginner/766.py
'''
The area of an equilateral triangle is ¼(√3a2) where "a" represents a side of the triangle. You are provided with the side "a". Find the area of the equilateral triangle.

Input Description:
The side of an equilateral triangle is provided as the input.

Output Description:
Find the area of the equilateral triangle and print the answer up to 2 decimal places after rounding off.

Sample Input :
20
Sample Output :
173.21'''
side = float(input())
area = (1 * (3 ** 0.5) * (side * side))/4
print(round(area,2))
